sonar-plugin-api/src/main/java/org/sonar/api/technicaldebt/batch/internal/DefaultRequirement.java

```diff
@@ -29,6 +29,7 @@
 import org.sonar.api.utils.internal.WorkDuration;
 
 import javax.annotation.CheckForNull;
+import javax.annotation.Nullable;
```

Comment:
:new: ![MINOR](https://raw.githubusercontent.com/henryju/image-hosting/master/minor.png)Remove this duplicated import.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)